File: CSharpPractice/LeetcodePractice/Leetcode530.cs
public class Solution530 {
    List<int> numList = new List<int>();
    public void helper(TreeNode node)
    {
        if(node==null){return;}
        numList.Add(node.val);
        helper(node.right);
        helper(node.left);
        return;
    }
    public int GetMinimumDifference(TreeNode root)
    {
        int ans=1000000;
        helper(root);
        numList.Sort();
        for(int i=1;i<numList.Count;i++)
        {
            int temp=numList[i]-numList[i-1];
            if(ans>temp){ans=temp;}

        }
        return ans;
    }
}

File: CppPractice/CppLeetcode/Leetcode530.cpp
class Solution {
public:
    vector<int> numList;
    void helper(TreeNode* node)
    {
        if(!node){return;}
        numList.push_back(node->val);
        helper(node->right);
        helper(node->left);
        return;
    }
    int getMinimumDifference(TreeNode* root) 
    {
        helper(root);
        int ans=100000;
        sort(numList.begin(),numList.end());
        for(int i=1;i<numList.size();i++)
        {
            if(numList[i]-numList[i-1]<ans)
            {
                ans=numList[i]-numList[i-1];
            }
        }
        return ans;
    }
};